packages/ember-intl/tests/unit/services/intl.js

```diff
@@ -1,75 +0,0 @@
-import { moduleFor } from 'ember-qunit';
-import test from 'dummy/tests/ember-sinon-qunit/test';
-
-let service;
-
-moduleFor('service:intl', 'Unit | Service | intl', {
-  setup() {
-    service = this.subject();
-  }
-});
-
-let message, translation, key, result, formatStub, findStub, formatArgs, findArgs;
-
-function setupT(options, formats) {
```

Comment:
tests were in dev awhile back but were never complete, now they're obsolete from 1.x and I need to rewrite them now that services/intl is running in the tests